### PR TITLE
ttyplot: update to 1.5.2.

### PR DESCRIPTION
**Pull request by MIvanchev**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-GLIBC)

### CODE REVIEW

File: srcpkgs/ttyplot/template [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-revision=1
+version=1.5.2
+revision=2
 build_style=gnu-makefile
 makedepends="ncurses-devel ncurses-libtinfo-devel"
 short_desc="Realtime plotting utility for terminal with data input from stdin"
 maintainer="misuchiru03 <misuchiru03+void@gmail.com>"
 license="Apache-2.0"
 homepage="https://github.com/tenox7/ttyplot"
 distfiles="${homepage}/archive/${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/ttyplot/template:11]:
```suggestion
distfiles="https://github.com/tenox7/ttyplot/archive/${version}.tar.gz"
```

Comment (MIvanchev) [reply srcpkgs/ttyplot/template]: Done!

**Comment by MIvanchev** [comment]
I'm removing the addition of the man page due to linter error. It will be added with another MR right after.

**Comment by classabbyamp** [comment]
why? the linter was only telling you not to increment revision because you were updating the package. you can still make changes like that.

revision is for when an update is *not* happening and a package should be rebuilt.

**Comment by MIvanchev** [comment]
A good to know, I was misunderstanding the docs then, sec.

**Comment by MIvanchev** [comment]
@classabbyamp do the checks start automatically, are they queued?

**Pull request merged**